presto-jdbc/src/test/java/io/prestosql/jdbc/TestPrestoDriverUri.java

```diff
@@ -213,6 +214,16 @@ public void testUriWithSslEnabledPassword()
         assertEquals(properties.getProperty(SSL_TRUST_STORE_PASSWORD.getKey()), "password");
     }
 
+    @Test
+    public void testUriWithExtraCredentials()
+            throws SQLException
+    {
+        String extraCredentials = "test.token.foo:bar;test.token.abc:xyz";
```

Comment:
also tests:
 - duplicates
 - credentials with same name and different values
 - invalid credentials like `invalid`,`invalid::`, invalid `invalid:;:;` or `invalid;;`